modules/gateways/callback/blockonomics.php

```diff
@@ -47,20 +47,26 @@
 
 $confirmations = $blockonomics->getConfirmations();
 
+$blockonomics_currency_code = $order['blockonomics_currency'];
+$active_currencies = $blockonomics->getActiveCurrencies()['currencies'];
```

Comment:
Instead of *getActiveCurrencies* this can use *getSupportedCurrencies* . *getActiveCurrencies* has overhead of extra DB call that is not required here.

Reply:
Fixed with dbcfd12